### ISSUE TITLE
A11y_Model Builder_Create New Experiment_Resize Text: When resized text to 200%, entire content in 'Create New Experiment' popup is not clearly visible.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1596]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11y_Model Builder; #A11yMAS; #A11yTCS; #WCAG1.4.4; #Win32; #DesktopApp; #A11ySev2; #ModelBuilder-Win32-Jun2021; #Visual Studio Client; 

### Environment Details:
Application name: Model Builder

### OS Details:
Windows Version 21H1 OS Build (19043.1083)
Microsoft Visual Studio

### Note:
This issue is observed in all popup screens.

### Repro Steps:

1. Change the display resolution average resolution (1280*800). Resize text to 200%.
2. Enable ML.NET Model builder as a preview feature in tools>options>Environment>preview features.
2. Create a .NET C# project and tab till project under solution explorer and Add machine learning.
3. Tab till Image classification under scenario tab and press enter.
4. Select Azure under environment tab and tab till set up workspace and press enter.
5. In 'Create New Experiment' dialog, verify whether text in all fields and create, cancel buttons are clearly visible or not

### Actual Result:
When resized text to 200%, entire content in 'Create New Experiment' popup is not visible.

### Expected Result:
When resized text to 200%, entire content in 'Create New Experiment' popup should be clearly visible. There should not be any loss of content.

### User Impact:
Low vision users who resize text for better visibility, will not be able to understand the content if text not clearly visible

**Comment by zewditu** [comment]
Issues tot reproducible.
 ## For this settings 
![image.png](https://images.zenhubusercontent.com/5d014b8c2b5aa11c6166d13d/792c3988-8487-428e-adc4-324499630059)

## All parts are visible.

![image.png](https://images.zenhubusercontent.com/5d014b8c2b5aa11c6166d13d/0a2fc706-e29b-465e-9cbf-2cadc1373353)

**Comment by NiharikaTipirishetty** [comment]
@zewditu For resize, please follow below steps:
1. please select resolution that is average of all display resolutions, (1280*720 in my machine) 
2. Navigate to 'Ease of access' settings, select display in sitemap. Under 'Make text size bigger' section, drag the slider to 200% and click on Apply button.

Issue is still reproducing in latest model builder version. 

https://user-images.githubusercontent.com/83570638/131449553-bd3842c9-1e7c-4803-9a71-6f237d3c5f25.mp4

**Comment by NiharikaTipirishetty** [comment]
@zewditu this issue is still reproducing at my end. all the controls are still not clearly visible at 200% zoom


https://user-images.githubusercontent.com/83570638/133403501-2e983fe8-85f4-4bd7-bcc1-af46cbdf0be0.mp4

**Comment by beccamc** [comment]
@NiharikaTipirishetty Sorry for the confusion on this one. 

It looks like this is working on 16.7.4.2146402.  Build available [here](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdevdiv.visualstudio.com%2F_apis%2Fresources%2FContainers%2F8681620%2FVSSetup_Release17%3FitemPath%3DVSSetup_Release17%252FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix&data=04%7C01%7Cbeccam%40microsoft.com%7C947a0dc64c814ae6a7e908d9782e36f0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637672962181378379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=PepyIHsbzrUWewREQwB9QN4MIm0Cws4cQOI7qMQvF9E%3D&reserved=0)

![image](https://user-images.githubusercontent.com/9122518/133515794-d64a960c-a546-499e-8034-1688d36ba229.png)

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. At 200% zoom, all controls in the Create new experiment, Create new workspace, Create new compute popups are clearly visible without any loss of content. Hence closing this bug.

![1596_fixed (2)](https://user-images.githubusercontent.com/83570638/134011270-91d5f653-c3b8-44f1-9182-5d3b1fc9e930.png)
![1596_fixed (3)](https://user-images.githubusercontent.com/83570638/134011276-af4cc68a-886b-4276-a3eb-8650a9b50ad6.png)
![1596_fixed](https://user-images.githubusercontent.com/83570638/134011280-9f6b7d09-f2e9-4b4b-aaf1-798b2fc7508c.png)

**Comment by VijayaBharathyA** [comment]
#Closed;
